### ISSUE TITLE
Error 'ipv4' when onboarding Juniper device

### ISSUE THREAD

**Issue by callumbnz** [nautobot/nautobot-plugin-device-onboarding#14]
### Environment
* Python version: 3.8.3
* Nautobot version:  1.0.2
* nautobot-device-onboarding version:  1.0.1

### Steps to Reproduce
1. Add device to plugin
2. fill out relevant details
3. click submit

Device be onboarded

Unsuccessful onboarding with error 'ipv4'

Jun 03 09:39:21 nautobot nautobot-server[65484]: 23:39:21 START: onboard device
Jun 03 09:39:21 nautobot nautobot-server[65484]: 23:39:21 Getting device with IP lookup failed: Device matching query does not exist.
Jun 03 09:39:21 nautobot nautobot-server[65484]: 23:39:21 CHECK: IP x.x.x.x:22
Jun 03 09:39:21 nautobot nautobot-server[65484]: 23:39:21 COLLECT: device information x.x.x.x
Jun 03 09:39:21 nautobot nautobot-server[65484]: 23:39:21 INFO guessing device type: x.x.x.x
Jun 03 09:39:29 nautobot nautobot-server[65484]: 23:39:29 INFO guessed device type: juniper_junos
Jun 03 09:39:29 nautobot nautobot-server[65484]: 23:39:29 Guessed Netmiko Device Type: juniper_junos
Jun 03 09:39:29 nautobot nautobot-server[65484]: 23:39:29 COLLECT: device facts
Jun 03 09:39:35 nautobot nautobot-server[65484]: 23:39:35 COLLECT: device interface IPs
Jun 03 09:39:36 nautobot nautobot-server[65484]: 23:39:36 INFO: No onboarding extension defined for napalm driver junos, using default napalm driver
Jun 03 09:39:36 nautobot nautobot-server[65484]: 23:39:36 Onboarding Error - Exception
Jun 03 09:39:36 nautobot nautobot-server[65484]: 23:39:36 'ipv4'
Jun 03 09:39:36 nautobot nautobot-server[65484]: 23:39:36 default: Job OK (fe066734-bfd3-418c-a418-38dd39594d8a)
Jun 03 09:39:36 nautobot nautobot-server[65484]: 23:39:36 Result is kept for 500 seconds

**Comment by mzbroch** [comment]
@callumbnz
Would it be possible for you to provide us with an output of your NAPALM driver's `.get_interfaces_ip()` method?

**Comment by callumbnz** [comment]
Ummm sure - if you can tell me how i do that no worries!

**Comment by NickKeating** [comment]
@mzbroch not sure if this is the output you are referring too, if not please let me know and I will remove the comment. I am getting the same error as Callum for this device when I try to onboard it. This is the result of direct NAPALM call to the same box.

```nautobot@nautobot:~$ 
nautobot@nautobot:~$ napalm wipm-e-501 --user d725702 --password ******* --vendor junos  --debug call get_interfaces_ip
2021-06-11 10:03:22,774 - napalm - DEBUG - Starting napalm's debugging tool
2021-06-11 10:03:22,774 - napalm - DEBUG - Gathering napalm packages
2021-06-11 10:03:22,775 - napalm - DEBUG - napalm==3.3.0
2021-06-11 10:03:22,775 - napalm - DEBUG - get_network_driver - Calling with args: ('junos',), {}
2021-06-11 10:03:22,776 - napalm - DEBUG - get_network_driver - Successful
2021-06-11 10:03:22,776 - napalm - DEBUG - __init__ - Calling with args: (<class 'napalm.junos.junos.JunOSDriver'>, 'wipm-e-501', 'd725702'), {'password': '*******', 'timeout': 60, 'optional_args': {}}
2021-06-11 10:03:22,777 - napalm - DEBUG - __init__ - Successful
2021-06-11 10:03:22,777 - napalm - DEBUG - pre_connection_tests - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>,), {}
2021-06-11 10:03:22,777 - napalm - DEBUG - open - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>,), {}
2021-06-11 10:03:23,397 - napalm - DEBUG - open - Successful
2021-06-11 10:03:23,397 - napalm - DEBUG - connection_tests - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>,), {}
2021-06-11 10:03:23,397 - napalm - DEBUG - get_facts - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>,), {}
2021-06-11 10:03:52,341 - napalm - DEBUG - Gathered facts:
{
    "vendor": "Juniper",
    "model": "MX960",
    "serial_number": "********",
    "os_version": "19.4R3.11",
    "hostname": "wipm-e-501-re1",
    "fqdn": "wipm-e-501-re1",
    "uptime": 1389390,
    "interface_list": [
        "gr-1/0/0",
        "ip-1/0/0",
        "lc-1/0/0",

 <<< OUTPUT CUT >>>
 
        "rbeb",
        "tap",
        "vtep"
    ]
}
{
    "vendor": "Juniper",
    "model": "MX960",
    "serial_number": "********",
    "os_version": "19.4R3.11",
    "hostname": "wipm-e-501-re1",
    "fqdn": "wipm-e-501-re1",
    "uptime": 1389390,
    "interface_list": [
        "gr-1/0/0",
        "ip-1/0/0",
        "lc-1/0/0",

 <<< OUTPUT CUT >>>

        "rbeb",
        "tap",
        "vtep"
    ]
}
2021-06-11 10:03:52,351 - napalm - DEBUG - get_facts - Successful
2021-06-11 10:03:52,351 - napalm - DEBUG - method - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>, 'get_interfaces_ip'), {}
2021-06-11 10:03:52,352 - napalm - DEBUG - get_interfaces_ip - Attempting to resolve method
2021-06-11 10:03:52,352 - napalm - DEBUG - get_interfaces_ip - Attempting to call method with kwargs: {}
2021-06-11 10:04:00,884 - napalm - DEBUG - get_interfaces_ip - Response
{
    "gr-1/0/0.0": {
        "ipv4": {
            "60.60.60.1": {
                "prefix_length": 24
            }
        }
    },
 

<<< OUTPUT CUT >>>

    "lo0.16384": {
        "ipv4": {
            "127.0.0.1": {
                "prefix_length": 32
            }
        }
    }
}
2021-06-11 10:04:01,138 - napalm - DEBUG - method - Successful
2021-06-11 10:04:01,138 - napalm - DEBUG - close - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>,), {}
2021-06-11 10:04:01,229 - napalm - DEBUG - close - Successful
2021-06-11 10:04:01,229 - napalm - DEBUG - post_connection_tests - Calling with args: (<napalm.junos.junos.JunOSDriver object at 0x7fea48525d30>,), {}
nautobot@nautobot:~$ 
```

**Comment by mzbroch** [comment]
Fixed in #16
